Cargo.toml

```diff
@@ -189,8 +189,8 @@ tempfile = "3.8.1"
 #
 # fork of `wasm-instrument`
 #
-# https://github.com/gear-tech/wasm-instrument/tree/v0.2.3-sign-ext
-gwasm-instrument = { version = "0.2.3", default-features = false }
+# https://github.com/gear-tech/wasm-instrument/tree/v0.3.0-sign-ext-utils
```

Comment:
btw can't we rename this branch to just 0.3.0 or gear-stable-0.3.0 etc?

Reply:
Makes sense, I can rename the branch after review gear-tech/wasm-instrument#7

Reply:
@grishasobol @ark0f please reflect on mentioned pr above

Reply:
I think we must relay only on branches, which updates only by approved PRs